ipatests/test_integration/test_adtrust_install.py

```diff
@@ -216,3 +220,79 @@ def test_schema_compat_attribute(self):
         value = (r'ipaexternalmember=%deref_r('
                  '"member","ipaexternalmember")')
         assert value in entry_list
+
+
+class TestIpaNisManage(IntegrationTest):
+    """
+    Tests for ipa-nis-manage CLI tool
+    """
+    @classmethod
+    def install(cls, mh):
+        tasks.install_master(cls.master)
+
+    def test_ipa_nis_manage_enable(self):
+        """
+        This testcase checks if ipa-nis-manage enable
+        command enables plugin on an IPA master
+        """
+        serverid = realm_to_serverid(self.master.domain.realm)
+        dirsrv_service = "dirsrv@%s.service" % serverid
+        console_msg = (
+            "Enabling plugin\n"
+            "This setting will not take effect until you restart Directory Server.\n"
+            "The rpcbind service may need to be started"
+        )
+        status_msg = "Plugin is enabled"
+        tasks.kinit_admin(self.master)
+        result = self.master.run_command(
+            ["ipa-nis-manage", "enable"],
+            stdin_text=self.master.config.admin_password,
+        )
+        assert console_msg in result.stdout_text
+        self.master.run_command(["systemctl", "restart", dirsrv_service])
+        time.sleep(DIRSRV_SLEEP)
+        self.master.run_command(["systemctl", "restart", "rpcbind"])
+        result = self.master.run_command(
+            ["ipa-nis-manage", "status"],
+            stdin_text=self.master.config.admin_password,
+        )
+        assert status_msg in result.stdout_text
+
+    def test_ipa_nis_manage_disable(self):
+        """
+        This testcase checks if ipa-nis-manage disable
+        command disable plugin on an IPA Master
+        """
+        serverid = realm_to_serverid(self.master.domain.realm)
+        dirsrv_service = "dirsrv@%s.service" % serverid
```

Comment:
These 2 lines are redundant of line 238, 239. Would be good idea to move this to method:
```
def get_dirsrv_id(realm):
    serverid = realm_to_serverid(realm)
    return("dirsrv@%s.service" % serverid)
```

Reply:
@mrizwan93 made the changes suggested.